docroot/modules/custom/sitenow_articles/src/Plugin/Block/GalleryJumpLink.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Drupal\sitenow_articles\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/**
+ * An image gallery jump link block.
+ *
+ * @Block(
+ *   id = "galleryjumplink_block",
+ *   admin_label = @Translation("Gallery Jump Link Block"),
+ *   category = @Translation("System")
```

Comment:
Is there a reason why we didn't use the `Site custom` category here?

Reply:
When we were setting it up, `System` felt like the closest fit for a block that's intended for use by developers creating layouts, and not for per-page custom layouts. We'd done a fair amount of going back and forth when deciding, and could be swayed.

Reply:
`Site custom` is what we have used for all site defined blocks. This includes placed blocks for AoS, UIPress and is then restricted using layout builder restrictions. Alternatively, we have a few blocks that use the `Restricted` category (Region Item, Search).

I feel like `System` should be left for core.

Reply:
That's fair re: leaving `System` to core.

`Site custom` to me sounds like blocks on an individual site like admissions, and not for blocks that may be used across the platform. Maybe `Restricted` then?

Reply:
+1 Restricted (forgot that this was default and not site specific)

Reply:
Sold. Moved it over.